source/instructors/setupcourses/library/global-library.rst

```diff
@@ -4,39 +4,93 @@
 .. _global-library:
 
 Codio Global Assessments Library
-================================
-The Global Assessments Library is an assessment library to which all Codio users have *read-only* access. The library contains auto-graded assessment questions that cover a wide variety of topics, difficulties, and assessment types. We are currently populating it with assessments in:
+=================================
+
+The Global Assessments Library is an assessment library to which all Codio users have **read-only access**. The library contains auto-graded assessment questions that cover a wide variety of topics, difficulties, and assessment types.
+
+Current Coverage
+----------------
+
+We are currently populating it with assessments in:
 
 - Java
 - C / C++
 - Python
 - JavaScript
 - Data Structures (in Java)
 
-All assessments in the global library are auto-graded, contain example solutions and answer explanations for the student, and some include teacher notes to help better convey the intended learning objective of the assessment.
+Assessment Features
+-------------------
+
+All assessments in the global library are auto-graded and contain example solutions and answer explanations for students. Some assessments also include teacher notes to help better convey the intended learning objective.
+
 
 Using the Global Assessments Library
 ------------------------------------
+
 To access the Codio Global Assessments Library, follow these steps:
 
-1. Click the **Assessments** button and choose **Assessment** from the **Add from Library** area
-2. On the **Choose an Assessment to Add** dialog, click the **Library Name** drop-down and choose **Codio Main**.
-3. You can filter through the different assessments by tags:
+1. From the course dashboard navigate to **Edit Assignments**. 
+2. Click and Open an Assignment. 
+3. Click **Edit**. 
+4. Click the **Assessments** button and choose **Assessment** from the **Add from Library** section.
+5. On the **Choose an Assessment to Add** dialog, click the **Library Name** drop-down and choose **Codio Main**.
+6. You can filter through the different assessments by tags:
 
-   .. image:: /img/bloomsTax.png
-      :alt: Bloom's Taxonomy
+.. image:: /img/bloomsTax.png
+   :alt: Bloom's Taxonomy
+   :width: 650px
   
-  - Programming language
-  - Assessment type (auto-detected)
-  - Category (topic-level)
-  - Content (sub-topic level)
-  - Learning Objective (in SWBAT form - "Students will be able to....")
-  - Bloom's Taxonomy level
+.. list-table:: Filter Categories and Inputs
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Category
+     - Available Inputs & Description
+   * - Bloom's Taxonomy Level
+     - .. tab-set::
+
+          .. tab-item:: Levels I-III
```

Comment:
These feel like they make more sense as a table rather than tabs but not a huge deal.